Code/Tools/ProjectManager/Platform/Mac/ProjectUtils_mac.cpp

```diff
@@ -18,44 +18,49 @@ namespace O3DE::ProjectManager
 {
     namespace ProjectUtils
     {
-        AZ::Outcome<QProcessEnvironment, QString> GetCommandLineProcessEnvironment()
+        AZ::Outcome<void, QString> SetupCommandLineProcessEnvironment()
         {
             // For CMake on Mac, if its installed through home-brew, then it will be installed
             // under /usr/local/bin, which may not be in the system PATH environment. 
             // Add that path for the command line process so that it will be able to locate
             // a home-brew installed version of CMake
-            QProcessEnvironment currentEnvironment(QProcessEnvironment::systemEnvironment());
-            QString pathValue = currentEnvironment.value("PATH");
-            pathValue += ":/usr/local/bin";
-            currentEnvironment.insert("PATH", pathValue);
-            return AZ::Success(currentEnvironment);
+            QString pathEnv = qEnvironmentVariable("PATH");
+            QStringList pathEnvList = pathEnv.split(":");
+            if (!pathEnvList.contains("/usr/local/bin"))
+            {
+                pathEnv += ":/usr/local/bin";
+                qputenv("PATH", pathEnv.toStdString().c_str());
```

Comment:
`qputenv` returns 0 if the variable could not be set and should likely be used for the return value.